website/pages/v0.8.3/endpoints.js

```diff
@@ -0,0 +1,26 @@
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary.js');
+const Container = CompLibrary.Container;
+
+function Endpoints() {
+    return (
+        <div className="separateOnPageNav">
+            <div className="docMainWrapper wrapper">
+                <Container className="mainContainer documentContainer postContainer">
+                    <div className="post">
+                        <link href="../../css/api.css" rel="stylesheet"></link>
+
+                        <div id="redoc"></div>
+                        <script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.41/bundles/redoc.standalone.js"></script>
+                        <script src="../../js/v0.8.3.js"></script>
```

Comment:
not really good at React, but I think we can create one `Endpoints` component and pass `src` to it.

If I'm right
```js
<script src="{props.src}"></script>
```

Reply:
I have created a `Endpoint` component and put it under `/website/static/js` directory. But when I try to import and use the component, its returning this error:
```
Error: Cannot find module '../../static/js/Endpoint'
```
Not really sure why it's not working